Tasks/PublishToPyPI/task.json

```diff
@@ -0,0 +1,54 @@
+{
+  "id": "2d8a1d60-8ccd-11e7-a792-11ac56e9f553",
+  "name": "PublishToPyPI",
+  "friendlyName": "Publish to PyPI",
+  "description": "Create and upload a Python package to a Python Package Index using twine.",
+  "author": "Microsoft Corporation",
+  "helpMarkDown": "[More Information](https://aka.ms/pypipublisher)",
+  "category": "Utility",
+  "visibility": [
+    "Build",
+    "Release"
+  ],
+  "demands": [],
+  "version": {
+    "Major": 0,
```

Comment:
If we are changing the name of the task, this is a breaking change for everyone's YAML.  Should we bump the major?

Reply:
It isn't a breaking change because build definitions and templates refer to the task by its ID (GUID), like [here](https://mseng.visualstudio.com/DefaultCollection/VSOnline/_git/VSO?path=%2FTfs%2FService%2FBuild2%2FServer%2FTemplates%2FNewCIWorkflow%2FPythonPackage.json&version=GBmaster&_a=contents&line=177&lineStyle=plain&lineEnd=178&lineStartColumn=1&lineEndColumn=1).  When S135 deploys, it would break YAML builds that use this task, but those should be very few and both YAML and this task are in preview.